eiffel-syntax-and-usage/glossary.md

```diff
@@ -49,7 +49,7 @@ An activity could also be an SCM operation such as a manual code review or autom
 
 Some activity types have their own specific events, such as EiffelTestCase\*Events notifying test case executions, while others are more generic and can be notified using EiffelActivity\*Events.
 
-#### Examples of events related to activities:
+__Examples of events related to activities:__
```

Comment:
This change would break the anchors and would break potential links/bookmarks that users have to that anchor.
[Takes you directly to Examples of events related to activities](https://github.com/eiffel-community/eiffel/blob/master/eiffel-syntax-and-usage/glossary.md#examples-of-events-related-to-activities)
[Does not take you directly to Examples of events related to activities](https://github.com/eiffel-community/eiffel/blob/241128574067aae54f354d3128808bd6528f1ee9/eiffel-syntax-and-usage/glossary.md#examples-of-events-related-to-activities)

Reply:
True, in [81b0e00](https://github.com/eiffel-community/eiffel/pull/304/commits/81b0e002645a968159e0eaabf2e6abba80d6f85d) I reverted to heading and instead removed trailing punctuation as the HTML link generator ignores those.